unoplat-code-confluence-docs/AGENTS.md

```
@@ -17,13 +17,40 @@
+### Build
+- `bun run build` (config: `package.json`, working directory: repo root)
+
+### Dev
+- `bun run dev` (config: `package.json`, working directory: repo root)
+
+### Test
+- Not detected
+
+### Lint
+- Not detected
+
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)